src/main/java/com/metamx/common/StringUtils.java

```diff
@@ -84,4 +84,24 @@ public static String safeFormat(String message, Object... formatArgs)
       return bob.toString();
     }
   }
+
+  // should be used only for estimation.. does not check validity of format
```

Comment:
Can you add javadoc that this is usually equivalent to `StringUtils.toUtf8(value).length`?